### PR TITLE
Use FILE_ALLOWED_EXTENSIONS to validate form upload documents

### PR DESCRIPTION
**Pull request by sandeepsajan0**
Fixes [hypha-issues/2](https://github.com/HyphaApp/hypha-issues/issues/2)

### CODE REVIEW

File: hypha/public/forms/models.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ class FormField(AbstractFormField):
 
 class ExtendedFormBuilder(FormBuilder):
     def create_document_field(self, field, options):
-        return FileField(**options)
+        return FileField(widget=FileInput(attrs={'accept': settings.FILE_ACCEPT_ATTR_VALUE}), **options)
```
Comment (frjo) [inline hypha/public/forms/models.py:45]: I changed to FILE_ACCEPT_ATTR_VALUE so the formatting gets correct for the accept attribute.

**Review by frjo — CHANGES_REQUESTED** [review]
This works nicely client side. We would need a check on the backend as well before saving the form.

**Comment by sandeepsajan0** [comment]
We are already having these validations on file fields(on both Multi and Single FileField). These are checking the forms before saving. https://github.com/HyphaApp/hypha/blob/main/hypha/apply/stream_forms/fields.py#L51

@frjo Where else do you think we need it?

**Comment by frjo** [comment]
The "Forms" are not using the uploads field from stream forms but a custom solution. So somewhere here maybe.

https://github.com/HyphaApp/hypha/blob/main/hypha/public/forms/models.py#L73-L93

**Comment by frjo** [comment]
Rebased to get in some fixes so test run as they should.

**Comment by sandeepsajan0** [comment]
@frjo I have tried to add validation in backend with custom clean method in form builder. Let me if it works good enough.

**Review by frjo — APPROVED** [review]

**Comment by Erioldoesdesign** [comment]
Can this be merged?

**Pull request merged**